c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -310,7 +310,7 @@ help:
 	@echo "Utility targets:"
 	@echo
 	@echo  "- update		Update the project (Git, Node, Pip)"
-	@echo  "- update-git-submodules	Update the Git submodules"
+	@echo  "- update-git-submodules	Update the Git, submodules, and clean unrevised file"
```

Comment:
better sentence:
`Update Git, the submodules, and clean unrevised files`
for me the meaning is different then `Update the Git submodule ...`!